examples/bri-3/src/bri/identity/bpiAccounts/api/accounts.controller.spec.ts

```diff
@@ -178,7 +188,7 @@ describe('AccountController', () => {
         bpiSubjectAccount.ownerBpiSubject.id,
       );
 
-      expect(bpiAccounts[1].id).toEqual(secondBpiAccountId);
+      expect(bpiAccounts[1].id).toEqual(secondBpiAccount.id);
```

Comment:
please change to assert object

Reply:
i tried once, not same objects so don't want to try and figure out since this is not part of this PR, if there is a need to fix these please open another issue to handle this